package.json

```diff
@@ -8,125 +8,125 @@
   },
   "dependencies": {
     "ajaxchimp": "^1.3.0",
-    "autoprefixer": "^6.7.6",
+    "autoprefixer": "^7.1.1",
     "awesome-phonenumber": "^1.1.3",
     "axios-mock-adapter": "^1.8.1",
-    "babel": "^6.23.0",
```

Comment:
yes, `babel` was moved to be `babel-core` a while back. the `babel` package ([here](https://github.com/babel/babel/tree/master/packages/babel)) is basically just a deprecation message.